src/js/rx/components/ErrorView.jsx

```diff
@@ -15,7 +15,7 @@ class ErrorView extends React.Component {
     let content;
     let alert;
 
-    if (some(errors, errorCodeIncludes(['RX135']))) {
+    if (some(errors, errorCodeIncludes(['RX135', 'VA900']))) {
```

Comment:
Could this array and the others used in the conditionals be given names and moved out of this function, eg. `unacceptedTermsErrors`? Will help with readability and editing